packages/parrot-fetch/src/index.js

```diff
@@ -20,7 +20,13 @@ const contextFetch = context.fetch.bind(context);
 export const PARROT_STATE = 'PARROT_STATE';
 
 export default function init(scenarios) {
```

Comment:
I wonder if it would be better to allow the context to be passed in as an argument

```
const defaultContext = typeof global === 'undefined' ? self : global;
...

function init(scenarios, context = defaultContext) {
...
}

init(scenarios, global.thunks)

Reply:
yah that's a good idea

Reply:
updated this in: 
https://github.com/americanexpress/parrot/pull/45/commits/5a69a14300fa75af83186c51ff79296377242a82